src/thunderbots/software/ai/hl/stp/evaluation/player.cpp

```diff
@@ -0,0 +1,16 @@
+#include "ai/hl/stp/evaluation/player.h"
+
+using namespace AI::HL::STP;
+
+
+bool Evaluation::playerOrientationWithinAngleThresholdOfTarget(const Point position,
+                                                               const Angle orientation,
+                                                               const Point target,
+                                                               Angle threshold)
+{
+    Point pass_direction       = (target - position).norm();
+    Point facing_direction     = Point(1, 0).rotate(orientation);
+    double direction_threshold = threshold.cos();
+
+    return facing_direction.dot(pass_direction) > direction_threshold;
```

Comment:
Could we add a few comments here explaining what we're doing? Took me a few minutes to figure it out, just want to make sure I understand it properly.

Reply:
A potentially more readable alternative here is just creating the orientation to the target (the orientation of the current `pass_direction` vector), and just checking the difference between that and the player orientation is less than the threshold.

eg `(orientation1 - orientation2).abs() < threshold`